packages/terra-site/webpack.config.js

```diff
@@ -12,6 +12,15 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const I18nAggregatorPlugin = require('terra-i18n-plugin');
 const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
 
+const threadLoaderRule = {
+  loader: 'thread-loader',
+  options: {
+    workerParallelJobs: 50,
```

Comment:
Just curious, how did we decide on these loader values for the options?

Reply:
I tweaked it until it seemed optimal